source/backend/api/Services/DocumentSyncService.cs

```diff
@@ -313,35 +316,35 @@ private void SyncDocumentTypeMetadataTypes(SyncModel model, IList<PimsDocumentTy
             }
         }
 
-        private async Task<ExternalBatchResult> AsyncLinkDocumentMetadata(DocumentTypeModel documentTypeModel, PimsDocumentTyp pimsDocumentTyp, IList<DocumentType> retrievedDocumentTypes, IList<MetadataType> retrievedMetadataTypes)
+        private async Task<ExternalBatchResult> AsyncLinkDocumentMetadata(Models.PimsSync.DocumentTypeModel documentTypeModel, PimsDocumentTyp pimsDocumentTyp, IList<Models.Mayan.Document.DocumentTypeModel> retrievedDocumentTypes, IList<MetadataTypeModel> retrievedMetadataTypes)
```

Comment:
oh, is this why? since there are two DocumentTypeModels side-by-side?

Reply:
thats right. It was kind of ambiguous